src/applications/ivc-champva/10-7959f-2/config/submitTransformer.js

```diff
@@ -54,7 +57,7 @@ export default function transformForSubmit(formConfig, form) {
       phone: transformedData.veteranPhoneNumber,
       email: transformedData.veteranEmailAddress,
     },
-    supportingDocs: transformedData.uploadSection,
+    supportingDocs: getObjectsWithAttachmentId(transformedData),
```

Comment:
Rather than hardcoding the config path to the (previously single) file upload, use this function that dynamically grabs any file that's been uploaded out of the form data.